lang-portal/backend-flask/routes/vocabulary.py

```
@@ -0,0 +1,93 @@
+from flask import request, jsonify, g
```

Ruff (0.8.2)

`flask.g` imported but unused

Remove unused import: `flask.g`

(F401)